docs/data-sources/imported_logical_device.md

```diff
@@ -0,0 +1,60 @@
+---
+# Documentation generated by "gen/generator.go"; DO NOT EDIT.
+# In order to regenerate this file execute `go generate` from the repository root.
+# More details can be found in the [README](https://github.com/CiscoDevNet/terraform-provider-aci/blob/master/README.md).
+subcategory: "L4-L7"
+layout: "aci"
+page_title: "ACI: aci_imported_logical_device"
+sidebar_current: "docs-aci-data-source-aci_imported_logical_device"
+description: |-
+  Data source for ACI Imported Logical Device
+---
+
+# aci_imported_logical_device #
+
+Data source for ACI Imported Logical Device
+
+## API Information ##
+
+* Class: [vnsLDevIf](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/vnsLDevIf/overview)
+
+* Supported in ACI versions: 1.0(1e) and later.
+
+* Distinguished Name Format: `uni/tn-{name}/lDevIf-[{ldev}]`
+
+## GUI Information ##
+
+* Location: `Tenants -> Services -> L4-L7 -> Imported Devices`
+
+## Example Usage ##
+
+```hcl
+
+data "aci_imported_logical_device" "example_tenant" {
+  parent_dn      = aci_tenant.example.id
+  logical_device = aci_l4_l7_device.example_in_another_tenant.id
+}
+
+```
+
+## Schema ##
+
+### Required ###
+
+* `parent_dn` - (string) The distinguished name (DN) of the parent object, possible resources:
+  - [aci_tenant](https://registry.terraform.io/providers/CiscoDevNet/aci/latest/docs/resources/tenant) ([fvTenant](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/fvTenant/overview))
+* `logical_device` (ldev) - (string) The distinguished name (DN) of the logical device to export.
+
+### Read-Only ###
+
+* `id` - (string) The distinguished name (DN) of the Imported Logical Device object.
+* `annotation` (annotation) - (string) The annotation of the Imported Logical Device object. This attribute is supported in ACI versions: 3.2(1l) and later.
+* `description` (description) - (string) A description of this configuration item.
```

Comment:
Can we change the description to say it's related to Imported Logical Device object?